### PR TITLE
Remove Deprecated configs 2.8

### PR DESCRIPTION
**Pull request by sivel**
##### SUMMARY
Remove or bump deprecated configs scheduled for removal in 2.8

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
many

##### ANSIBLE VERSION

```paste below
2.8
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/config/base.yml [hunk 1]
```diff
@@ -392,7 +392,7 @@ DEFAULT_ASK_SUDO_PASS:
   default: False
   deprecated:
     why: In favor of Ansible Become, which is a generic framework. See become_ask_pass.
-    version: "2.8"
+    version: "2.9"
```
Comment (acozine) [inline lib/ansible/config/base.yml:395]: For modules we use `removed_in`, which is clearer. In this context `version` could be the version in which the term is deprecated, or the version in which it will be removed. The terminology pre-dates this PR, so it shouldn't block merging, but something to think about. See https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html.

File: lib/ansible/config/base.yml [hunk 2]
```diff
@@ -405,7 +405,7 @@ DEFAULT_ASK_SU_PASS:
...
```
Comment (acozine) [inline lib/ansible/config/base.yml:408]: Is this part of a larger conversation about the schedule for deprecating config options? I'm curious about why some are getting moved from 2.8 to 2.9, while others are getting moved from 2.8 to 2.12. If there's an issue somewhere that documents those decisions, a link to it would be helpful for context.

**Review by acozine — COMMENTED** [review]
Overall looks fine. It sparked a couple of questions, but no blockers.

File: lib/ansible/config/base.yml (hunk 2 above)
Comment (sivel) [reply lib/ansible/config/base.yml:408]: This is in part due to the inconsistency of how we handled deprecations in the past.

Starting with 2.7 I have been making an effort to get things "in order".  With 2.7 we bumped the deprecation of 2 features to future versions.

* su/sudo was bumped to 2.9 to coincide hopefully with the addition of become plugins
* include was bumped to 2.12, because we had some snafus and the replacement wasn't stable

It just so happened, we had a single feature, that had multiple deprecation versions, this is part of an ongoing effort to align our deprecations.

File: lib/ansible/config/base.yml (hunk 1 above)
Comment (sivel) [reply lib/ansible/config/base.yml:395]: I agree, but wasn't planning on changing them as part of the version bump.  FWIW, the code-smell test I am working on will look at both `version` and `removed_in`.  They are both supported, and have the same meaning in this context currently.

**Comment by acozine** [comment]
I'll leave the conversations "unresolved" just to keep them visible. But +1 to merging this.

File: docs/docsite/man/ansible-playbook.1.html [hunk 3]
```diff
@@ -1,31 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gundalow) [inline docs/docsite/man/ansible-playbook.1.html:1]: This change seems unrelated, and not sure why it's been removed

Comment (sivel) [reply docs/docsite/man/ansible-playbook.1.html:1]: These files are unused, but contain reference to env vars that are scheduled to be removed.

These files were added in https://github.com/ansible/ansible/commit/d6e4b1bf5b5d573c4f0161aec6d2235670fe41c2 on Oct 8, 2012 and have never been updated.

**Review by bcoca — APPROVED** [review]
